testsuite/regressiontests/ticket-507.sli

```diff
@@ -22,12 +22,12 @@
 
 /** @BeginDocumentation
 
-Name: testsuite::ticket-507 - Ensure that spike_detector throws exception on SetStatus if off-grid times are set
+Name: testsuite::ticket-507 - Ensure that spike_recorder throws exception on SetStatus if off-grid times are set
```

Comment:
I think the description was wrong before. Looking at the [test](https://github.com/nest/nest-simulator/blob/master/testsuite/regressiontests/ticket-507.sli) it seems like it should be `spike_generator` and `precise_times` and not `spike_recorder/spike_detector` and `off-grid` times..

Reply:
I've adapted the wording and corrected the mistake.